dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json

```diff
@@ -0,0 +1,605 @@
VmVFtjxZePGFHz2AVuFjhKusmIgvL07Tyr/fFTEN92oBI0Wjg3YkkDQzoTQhRGs9PW9JqXg3EjEkg7WzRtUAREchPKtRALkiO6H79gZaFgvCUq8QmJELOWE/CVrgnRQMWwIULM4LeoOW5rNyw4Fr8G6xiEJIhZyzhprwuI4iM8QqKB2j1GjD7tq5DDJiRAPCmB6xX6gNiEJIiH+4YGOUASP2ETEiDmdgUNo80CkvgJnxAfkW1vnU2TdqTYiIAQ36Nud2gYCy6TgEgIBTGH1yi3MDYmhEXxEREhsiJmq8iIcjImJCTEReQWMmMkY0JSQlxEzUhqkBITYiKiYdpOZpCSE+Ihcm3GSoQQeW7HcoI+iQ2I51E5i0kGMNduOQsn8qyaECIDi5LNeVzWnPbVPC+2qZO110gNLk9hdAkR5nK5jP1n+8P+uf27fcF+9/g7+9cPbx3+2v/juPaggma7p8fLZTTr1XshoVN7Ung3xM/8PYBRzO176ejQvjO7Vncd2r7J5MCUyS7qKqPXF9rpq1yh4ruPR+6PHTS8sp7pjFkLKMOJsbvFZKAA2cLQ2PTbHGcPB81oX62g5Ty9tT4xozLeGFvuVOEQM4wFg8hp7cNQlAePWzNmsplm2C4lvbYvfCtPxwWV2b1/NTweoIXiIQRfZuEo3ulj5D4zXCHSLiz12x6ggjB2/HLNj5AXC3F61kY5DTlfbnFWRK8vMo/tqHcGNqpPwmrmetGsLAGsiHIag5QQBVWvB4qrw3r0Pbv6fOKxdtP6OFs8jY2I8lLmi2yYclylT+spf+ps5bP4Frl3X4y9bEgE+Hy2qgQpc+I9VKMibrJhNMcnIAzdgkQouUFixe0cn8zXaDFvwp2s5vE+0PdZpIqEuLuTxmhzBWxpscZowx5z77Eeyt1QeG+J4Z3jdrU2ZlDI4RNGv49DNlcju2vCqKJG900zP8JwxFxhP2PTKxl8wsgGMSe8sOubsJwbUT9W8yUMRcwcn5XVt7ARo94qbiDHf9I5FOAjXsABhCGImZOt+3IXF7DgWqUzfPo5+z4YcIf4UPGzvupg/xB88MIRT9aeGMPCvRK5k21y+mxVnXj1ufNnedZPQZn+eEaaSbVymu29BxEGIWZO1w8Z5hXXnZ4QdqoPKC+uniWjam3t3UthPTkbqp0qysGntaOHXgUSBiByrkeb416Jmfc9x9jO+1H3+B9HANFnPnw2orB2DlX1h98e5OGQhy8DAf0RM651fKahERHqc0HZgQgrx+Dr/BdkBGHp+OXGYTT/rnffgfRfCKEf4rmTH+ON08zC5mucOEph7RinnQBfiDR1Ek6PhxF+TJtxFEbojciN3YCM+YjnbDQE8eNwlEvH2TOnvl2ztTJ8CO1TNTcZNZyQfzrvVe7x3IvJmM5GMydT5zicOgnXgb37cFy0DdfpqM58k7ZgRM7r6abmF54Rz9z5Ep9w5n57uYoAeI6Y+/IMRHXMmCg6W1OcniYWYUsIjg1REXPeW5NVTGdTEB+OTQnCxDk+1I+gnglj59zEqEYZlWGI3NhnxtPCzWz2VkRxbOyO4Wpg2qVM3NdLZ81j2fHEo+b89nkzDPYcQ9x0Slh+n+eacQlR2obHeETM+D+x3cROT0W74Fn1mi/EJ2RMlOZhMO4Rc0F36A2xZ/uiMp14Hlj/iU3IyJ0Kjhl3iFzQY6N03PSU07YZiEdvgyZ4vOKuA++kNla/8Rk3iJnA+56YFl56yrX7PodVa9PAvPSs1H344Nw/Yw9E1EKeqPSFNU5zVR/AWXA0RLqv+lw1arRg79QTCvbBgIyJZ8Sc6LVG0XmPMtR+a17Rq/WLAYgGRWgBV65hIo7O+mkG+pjTXk1nZ3dj1PAAeaUWWsYzF5gRQ1k1ToeqMZ9GTk8E/tvxWTQbw8ze+HV4DV5u4gEixOXEOBzF3moRo2PC/c9xmJu1sIm9v+YRKrH4JXCR3+c0aMYYV4o9G9yoUcU1IC98G75cJxpYuFYUhWkNjTYjOEL49Y5fzWR71Rvnwzst/WKPU/oYuz6MkrdVY46ZO28iR2eF+2FbZ3m/j/rYRtzOMgi6SKZp5EWUH/yVGjG8H/T0ExWQUfGNmCKiMI2x0tfErJ5uER/Ce0NDv3HWXuUFPmHhiX9IQ/w61oaJvvLPIf7G+yYIuRJhEdkfMQVAYRxzz4tJAJgGIs/H3RQiN/FXvtNAFGJu8bCN+IwfFAuJX4v8CmNfT/2RhDBZRP4+WkLqlDomMmKiiK7FgahqEGQ2ySK6lruiqyI+Eunp4T4Z8X/wABnzsUCImAzgwz329sGJQkaYEKJQwwVkGFIjJoLI3+MDMi3CK/GxkMC1KOFEir3kNikifXcjfRBtwu4Qj1PaiA9Lou8KYuQhqbOhjSiEF7mDZX4Rj1Oq7uYhQpE7RE1yQpqIPImb2cogdzYUEfkPg5iQaVnEzobetTiF+I51fUwOSMuKD39Avmu1Y10s4pQsUhxUA7gSqSBK0YvcwdJBCCkgSmDfzN0kD/s0EKX41SdfAYR9eERpDQfIdGAIYRElIDezkUxa0KCAKP0BvbETIj2FRQRISB2SJ0CEYIhSlF0XcWRWLwtR8tsshq86ECAUIvmcwi19DGVECETpAyzYH9UCSU+BEHGL3MH6ATMiMaKEW+QOlgkGSIy4pHSPPHEJHArx1/M+TgDpFtw4JUGUphTczFaNERwhAeIvFTezkQwXMQgQCYvcwYJ0NtiIU5qP4tBhChpEiFKN2lVoy3xPG1HC2XURR3NIQCxEwNKFp1SwOQYeovRN84E/Gw2WoIRxEafwc4ozgUaMuIjSB31AxoAqaGAgSmsjAUKmv0wLUVrSy2Ycak3TQZSSuAjTRJRAltIiqgMcMx4jLC9K35AV4FCpNTHZuChJnncp0hRw/haIKEnUczVvxtr71/JRhJIgeet+ul7V0uCzJXdmtQqY/nhrMutQnA9eddVVV1111VVXXXVVMvofAlT4ioxrICQAAAAASUVORK5CYII=)"
```

Comment:
Hey @jcountsNR, 
I don't think anyone has tried tossing a base64 image in any quickstart dashboards, I just want to confirm that you see this working on your end. If so, this is a great way to manage dashboard images!

Reply:
![Screen Shot 2022-09-23 at 2 27 59 PM](https://user-images.githubusercontent.com/94138069/192059060-af26171e-176e-485f-af9a-33c5d540f116.png)
Here is a screenshot, it's been working for me.